content/posts/November-2021.md

```diff
@@ -48,3 +48,23 @@ weight: 69
   needs to be present in the dist-git for now
   (the state for the new commit is used).
   ([packit-service#1278](https://github.com/packit/packit-service/pull/1278))
+
+## Week 48 (November 29th - December 3rd)
+
+- `oraclelinux-8` distro specification is now mapped to Oracle-Linux-8.5
+  compose.
+  ([packit-service#1301](https://github.com/packit/packit-service/pull/1301))
```

Comment:
I'd omit this, it's just for the internal testing farm (which only 2 projects use) and it's a tiny detail.

Reply:
Good point, I was a bit torn whether to include it or not but since it's so tiny, I've dropped it

Reply:
I find it funny that our release notes mention oraclelinux, so I'd keep it for sake of amusement :)